CMakeLists.txt

```diff
@@ -289,7 +289,7 @@ include(cmake/common/advss_helpers.cmake)
 setup_obs_lib_dependency(${LIB_NAME})
 setup_obs_lib_dependency(${PROJECT_NAME})
 
-find_package(Qt6 REQUIRED COMPONENTS Widgets Core)
+find_package(Qt6 REQUIRED COMPONENTS Widgets Core DBus)
```

Comment:
This will have to be moved into the the `# --- Linux-specific build settings and tasks ---` block as otherwise this will lead to configure errors on Windows / MacOS.